helpdesk_mgmt_ticket_closed_date/views/helpdesk_ticket_closed_date_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <record id="helpdesk_ticket_view_form_closed_date" model="ir.ui.view">
+        <field name="name">helpdesk.ticket.view.form (closed_date)</field>
+        <field name="model">helpdesk.ticket</field>
+        <field name="inherit_id" ref="helpdesk_mgmt.ticket_view_form"/>
```

Comment:
This view removes/replaces two field attributes (`readonly` and `attrs`), so it's a little bit dangeours. In those cases, the best practice is assigning a lower priority. Could you add it?